### PR TITLE
adding csi times and renaming the test

### PR DESCRIPTION
**Pull request by ypersky1980**
This PR contains the following enhancements: 

1) Renaming the test so that similarly to bulk pod attach and pod reattach test this test will be called "pod attach" and not "pvc attach" test

2) Adding CSI measurements for pod attach action and reporting those measurements to the ES. 

Signed-off-by: Yulia Persky <ypersky@redhat.com>

### CODE REVIEW

**Review by Avilir — DISMISSED** [review]

File: ocs_ci/helpers/performance_lib.py [hunk 1]
```diff
@@ -748,3 +752,92 @@ def wait_for_resource_bulk_status(
...
+        Exception : in case that the expected time logs are not found
+
+    """
+    volume_handle = None
+    for line in run_oc_command(f"describe pv {pv_name}", namespace=namespace):
+        if "VolumeHandle:" in line:
+            volume_handle = line.split()[1]
+            break
+    if volume_handle is None:
+        raise Exception("Can not get volume handle")
```
[the remark this answers is not in the document]
Comment (ypersky1980) [reply ocs_ci/helpers/performance_lib.py]: Added here and in similar places.

**Comment by ypersky1980** [comment]
Please note the measurements taken by the test: 

22:12:33 - MainThread - test_pod_attachtime - INFO  - Interface: CephBlockPool, PVC size: 5 GB. pod number 1 start time: 6 seconds, csi start time is 0.31539900000000004
22:12:33 - MainThread - test_pod_attachtime - INFO  - Interface: CephBlockPool, PVC size: 5 GB. pod number 2 start time: 9 seconds, csi start time is 0.185779
22:12:33 - MainThread - test_pod_attachtime - INFO  - Interface: CephBlockPool, PVC size: 5 GB. pod number 3 start time: 8 seconds, csi start time is 0.158137
22:12:33 - MainThread - test_pod_attachtime - INFO  - Interface: CephBlockPool, PVC size: 5 GB. pod number 4 start time: 9 seconds, csi start time is 0.175396
22:12:33 - MainThread - test_pod_attachtime - INFO  - Interface: CephBlockPool, PVC size: 5 GB. pod number 5 start time: 8 seconds, csi start time is 0.158057
22:12:33 - MainThread - test_pod_attachtime - INFO  - Interface: CephBlockPool, PVC size: 5 GB. The standard deviation percent for csi start time of the sampled 5 pods is 33.432320075105174

22:12:33 - MainThread - test_pod_attachtime - INFO  - Interface: CephBlockPool, PVC size: 5 GB. The average time for the sampled 5 pods is 8 seconds.
22:12:33 - MainThread - test_pod_attachtime - INFO  - Interface: CephBlockPool, PVC size: 5 GB. The average csi time for the sampled 5 pods is 0.1985536 seconds.

====================================================================


22:20:21 - MainThread - test_pod_attachtime - INFO  - Interface: CephFileSystem, PVC size: 5 GB. pod number 1 start time: 4 seconds, csi start time is 0.126086
22:20:21 - MainThread - test_pod_attachtime - INFO  - Interface: CephFileSystem, PVC size: 5 GB. pod number 2 start time: 9 seconds, csi start time is 0.07191399999999999
22:20:21 - MainThread - test_pod_attachtime - INFO  - Interface: CephFileSystem, PVC size: 5 GB. pod number 3 start time: 8 seconds, csi start time is 0.052207
22:20:21 - MainThread - test_pod_attachtime - INFO  - Interface: CephFileSystem, PVC size: 5 GB. pod number 4 start time: 8 seconds, csi start time is 0.043347000000000004
22:20:21 - MainThread - test_pod_attachtime - INFO  - Interface: CephFileSystem, PVC size: 5 GB. pod number 5 start time: 8 seconds, csi start time is 0.044846000000000004
22:20:21 - MainThread - test_pod_attachtime - INFO  - Interface: CephFileSystem, PVC size: 5 GB. The standard deviation percent for csi start time of the sampled 5 pods is 51.08794404778807


22:20:21 - MainThread - test_pod_attachtime - INFO  - Interface: CephFileSystem, PVC size: 5 GB. The average time for the sampled 5 pods is 7.4 seconds.
22:20:21 - MainThread - test_pod_attachtime - INFO  - Interface: CephFileSystem, PVC size: 5 GB. The average csi time for the sampled 5 pods is 0.06768 seconds.

File: ocs_ci/helpers/performance_lib.py [hunk 2]
```diff
@@ -748,3 +756,93 @@ def wait_for_resource_bulk_status(
     err_msg = f"{resource.upper()} failed reaching {status} on time"
     logger.error(err_msg)
     raise Exception(err_msg)
+
+
+def pod_csi_time(interface, pv_name, start_time, namespace=constants.OPENSHIFT_STORAGE_NAMESPACE):
```
Comment (yati1998) [inline ocs_ci/helpers/performance_lib.py]: I would prefer pod_attach_csi_time than pod_csi_time.

File: ocs_ci/helpers/performance_lib.py [hunk 3]
```diff
@@ -748,3 +756,93 @@ def wait_for_resource_bulk_status(
...
+    """
+    Get the starting/ending creation time of a PVC based on provisioner logs
```
Comment (yati1998) [inline ocs_ci/helpers/performance_lib.py:761]: The comments still says that the function calculates the creation time of PVC? Can you check this again?

**Review by yati1998 — CHANGES_REQUESTED** [review]

**Review by Avilir — DISMISSED** [review]
LGTM

**Review by yati1998 — APPROVED** [review]

**Review by ramkiperiy — APPROVED** [review]

**Review by Avilir — APPROVED** [review]
LGTM

**Review by pintojoy — APPROVED** [review]
LGTM

**Pull request merged**